### ISSUE TITLE
reference note from changeset comment

### ISSUE THREAD

**Issue by nebulon42** [openstreetmap/openstreetmap-website#874]
It would be great if one could reference a note in a changeset comment (e.g. by writing #NOTE_NR) and there would be automatically a link pointing to that note. Actually this would be quite similar as how it works on Github, though no need for the fancy popup-select-note-by-name stuff.

Example for a changeset where I fixed a note and referenced it: http://www.openstreetmap.org/changeset/28195830

**Comment by pnorman** [comment]
I'm not sure - we have a lot of objects with numeric IDs. I frequently reference changeset IDs in changeset comments. Maybe auto-linking "note N" where N is the ID of a note that exists at the time of changeset creation would be better.

**Comment by tomhughes** [comment]
I thought there was already a ticket for this (in a more general form that resolves the issues @pnorman refers to) but I can't find it right now...

**Comment by nebulon42** [comment]
Yes, I forgot to take other numbers as changeset ids into account. So a more general solution is needed.
@tomhughes I searched but did not find anything open, so maybe close as duplicate.

**Comment by emacsen** [comment]
I think there's a branch Lukasz has that addresses this for all object types.
